subsys/spm/spm.c

```diff
@@ -182,7 +188,7 @@ static void spm_config_flash(void)
 
 		NRF_SPU->FLASHREGION[i].PERM = flash_perm[i];
 
-		PRINT("%02u 0x%05x 0x%05x \t", i, 8 * KB(i), 8 * KB(i + 1));
```

Comment:
Where the other patches looked to just give names to hardcoded values. This patch looks like is making a semantic change.

Should this patch perhaps have it's own commit with a commit message explaining the change?

Reply:
Will create separate PR